src/main/java/vg/civcraft/mc/civchat2/CivChat2Manager.java

```diff
@@ -346,6 +368,22 @@ public void sendGroupMsg(Player sender, Group group, String message) {
 				sender.sendMessage(String.format(ChatStrings.globalMuted, TextUtil.formatDuration(mutedUntil - System.currentTimeMillis())));
 				return;
 			}
+			if(slowmodeTime != 0 && !sender.hasPermission("civchat2.globalslowmode")) {
```

Comment:
Common pattern would be to assume values less than 0 as slow mode being disabled